### ISSUE TITLE
bootstrap version is wrong?

### ISSUE THREAD

**Issue by snowpine** [nakupanda/bootstrap3-dialog#129]
in bower.json：
 
    bootstrap version should be： >= 3.1.1?
    Now is ~3.1.1，which is equivalent to 3.1.1~3.2.0 .

**Comment by nakupanda** [comment]
Hi @bitclaw ,

I'm still new to bower, can you please take a look at this? 

Thank you very much :)

**Comment by snowpine** [comment]
I installed it, but wrong：
  
    - Installation request for bower-asset/bootstrap-dialog >=1.34.2 -> satisfiable by bower-asset/bootstrap-dialog[v1.34.2].
    - bower-asset/bootstrap-dialog v1.34.2 requires bower-asset/bootstrap ~3.1.1 -> no matching package found.

**Comment by avdv** [comment]
I don't know wether one is notified about comments for specific commits, but I left one here: https://github.com/nakupanda/bootstrap3-dialog/commit/9b7745e33fb9644b5b475f546dcff5de724cbf54#commitcomment-10661710

tldr; the required bootstrap version in bower.json should be changed to reflect the supported versions, e.g. `3.1.1 - 3.3.4`.

**Comment by nakupanda** [comment]
Hi Guys,

My bad :(

 I'm going to change this next week.

**Comment by nakupanda** [comment]
Hi,

Thank you for reporting this, required bootstrap version has been updated to latest.

**Comment by snowpine** [comment]
@nakupanda Thanks!

**Comment by nakupanda** [comment]
I'm sorry for doing this too late, if you find anything abnormal and you have a good solution, please submit a pull request.
